README.md

```diff
@@ -158,6 +159,33 @@ $ java -jar xpanse-runtime-1.0.0-SNAPSHOT.jar --spring.profiles.active=zitadel \
 --authorization-swagger-ui-client-id=221664356859969539@eclipse-xpanse
 ```
 
+##### Execute authenticated APIs
+
+When the application activated authorization with 'zitadel', execute authenticated APIs must add a header
+'Authorization' with value like follows 'Bearer ${access token}' in the http request. Here are two ways to get
+access_token for executing authenticated APIs:
+
+The first way, open the openAPI page 'http://localhost:8080/swagger-ui/index.html' of the ran Xpanse service in browser.
```

Comment:
The first way, is to open the Swagger-UI page of the Xpanse server. In case of local development server, the Swagger-UI is reachable on `http://localhost:8080/swagger-ui/index.html`.

Reply:
fixed.